DEHPCommon/UserInterfaces/ViewModels/Interfaces/IObjectBrowserViewModel.cs

```diff
@@ -33,7 +33,7 @@ namespace DEHPCommon.UserInterfaces.ViewModels.Interfaces
     /// <summary>
     /// Interface definition for <see cref="ObjectBrowserBaseViewModel"/>
     /// </summary>
-    public interface IObjectBrowserViewModel
+    public interface IObjectBrowserViewModel : IDisposable
```

Comment:
Not sure this is not a non-breaking change. Adapter usage might be dependent on this. @nathanatrhea maybe a comment?

Reply:
It shouldn't make a difference by making the IObjectBrowserViewModel Inheriting the IDisposable instead of the ObjectBrowserBaseViewModel which anyway implements the IDisposable in the end. What I want to make sure actually is that autofac is still able to resolve the ObjectBrowserViewModel as IObjectBrowserViewModel.

Reply:
I just tested this branch with the ecosim adapter => no problem